crates/core/src/client/mod.rs

```diff
@@ -417,7 +417,7 @@ impl KakarotEthApi for KakarotClient<JsonRpcClient<HttpTransport>> {
                             )))?;
 
                             let evm_address = Felt252Wrapper::from(*evm_address);
-                            Some(H160::from(evm_address))
+                            Some(evm_address.try_into()?)
```

Comment:
removed h160 because its the implementation detail, Address is the most meaningful type

Reply:
yes for sure